### ISSUE TITLE
GQL - Type error on auto complete field

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#408]
**Description**
When using Google Places Autocomplete field for an address field the following console error is thrown when we try to call it in a GraphQL query

<img width="1325" alt="Screen Shot 2021-03-23 at 9 56 05 am" src="https://user-images.githubusercontent.com/21274925/112070719-2eca7780-8bbe-11eb-83dd-e02851e9f80c.png">

Sorry for all the issues again, I believe we're nearly done with the project now 😅 

**Steps to reproduce**
1. Create a form with an address field that only uses Autocomplete
2. Try to use it in a GQL query

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.24
- Craft version: 3.6.10
- Multi-site: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Can you shoot through the query you're using to fetch the address field?

And no need to apologise, thanks for reporting!

**Comment by d-karstens** [comment]
Sure! 

This is the query, with `locationFound` and `pickupLocation` being autocomplete fields coming back as empty strings

```graphql
query CouncilPets {
  submissions(form: "councilFoundPetSubmission") {
    ... on councilFoundPetSubmission_Submission {
      animalType {
        ... on animalTypes_Category {
          animalTypeGrouping
          id
          title
        }
      }
      id
      petName
      petStatus
      locationFound
      pickupLocation
    }
  }
}
```

I've had another look at the docs and seen the auto complete field is used to populate the other address fields, is this happening because we're using the auto complete without any of the other fields?

**Comment by engram-design** [comment]
Yep, just diving into it now, and it's a bit of an odd field option for headless. So I believe I left it out of GQL because it wasn't really needed, as you're providing your own front-end, hence the lack of a need to have this sort of a field (because you can DIY). Typically, you'd store the individual part (address/city/etc) of the address, but I can see you might also just want to store the entire address as-is, so we shouldn't prevent you from doing that.



Added for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.25",
  "...": "..."
}
```

Then run `composer update`. You should be able to do this:

```
mutation saveSubmission($addressAutocomplete:gql_addressAutocomplete_FormieAddressInput) {
    save_gql_Submission(addressAutocomplete: $addressAutocomplete) {
        addressAutocomplete
    }
}

{
    "addressAutocomplete": {
        "autocomplete": "42 Wallaby Way Sydney 2000 NSW Australia"
    }
}
```

**Comment by d-karstens** [comment]
Works perfect, thank you!

**Comment by engram-design** [comment]
Fixed in [1.3.26](https://github.com/verbb/formie/releases/tag/1.3.26)
